src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -22,7 +22,6 @@
         {%- if form is rootform -%}
             {{ form_errors(form) }}
         {%- endif -%}
-        {{- block('form_rows') -}}
```

Comment:
this is a BC break in case someone overwrites the block

Reply:
For this reason I liked https://github.com/symfony/symfony/pull/27247#discussion-diff-188221848R401 solution more. That would be BC break too in case someone overrides FormRenderer (and expect it to be called for already rendered field too), but that is much less likely imho